4.x: java.util.UnknownFormatConversionException: Conversion = 'F'

## Environment Details
* Helidon Version: 4.2.1
* Helidon SE

----------

## Problem Description

```
WARNING: Internal server error
io.helidon.http.RequestException: Internal error
	at io.helidon.http.RequestException$Builder.build(RequestException.java:157)
	at io.helidon.webserver.http1.Http1Connection.handle(Http1Connection.java:249)
	at io.helidon.webserver.ConnectionHandler.run(ConnectionHandler.java:175)
	at io.helidon.common.task.InterruptableTask.call(InterruptableTask.java:47)
	at io.helidon.webserver.ThreadPerTaskExecutor$ThreadBoundFuture.run(ThreadPerTaskExecutor.java:240)
	at java.base/java.lang.VirtualThread.run(VirtualThread.java:329)
Caused by: java.util.UnknownFormatConversionException: Conversion = 'F'
	at java.base/java.util.Formatter$FormatSpecifier.conversion(Formatter.java:2958)
	at java.base/java.util.Formatter$FormatSpecifier.<init>(Formatter.java:2994)
	at java.base/java.util.Formatter.parse(Formatter.java:2849)
	at java.base/java.util.Formatter.format(Formatter.java:2774)
	at java.base/java.util.Formatter.format(Formatter.java:2728)
	at java.base/java.lang.String.format(String.java:4390)
	at io.helidon.common.socket.SocketContext.message(SocketContext.java:102)
	at io.helidon.common.socket.SocketContext.log(SocketContext.java:93)
	at io.helidon.webserver.http.ErrorHandlers.handleRequestException(ErrorHandlers.java:147)
	at io.helidon.webserver.http.ErrorHandlers.unhandledError(ErrorHandlers.java:198)
	at io.helidon.webserver.http.ErrorHandlers.lambda$handleError$1(ErrorHandlers.java:178)
	at java.base/java.util.Optional.ifPresentOrElse(Optional.java:198)
	at io.helidon.webserver.http.ErrorHandlers.handleError(ErrorHandlers.java:177)
	at io.helidon.webserver.http.ErrorHandlers.runWithErrorHandling(ErrorHandlers.java:113)
	at io.helidon.webserver.http.Filters.filter(Filters.java:89)
	at io.helidon.webserver.http.HttpRoutingImpl.route(HttpRoutingImpl.java:74)
	at io.helidon.webserver.http1.Http1Connection.route(Http1Connection.java:544)
	at io.helidon.webserver.http1.Http1Connection.handle(Http1Connection.java:224)
	... 4 more
```

`HttpPrologue.toString()` is passed to the log format: https://github.com/helidon-io/helidon/blob/649e6c05f79c313143178ddde9f907045184c699/webserver/webserver/src/main/java/io/helidon/webserver/http/ErrorHandlers.java#L147

## Steps to reproduce

This can happen if the prolog contains contain URI encoded characters.
See the prologue below (from `org.eclipse.microprofile.jwt.tck.container.jaxrs.JsonValueInjectionTest#verifyIssuerClaim2`)

```
GET /endp/verifyInjectedIssuer?iss=https%3A%2F%2Fserver.example.com&auth_time=1747335396 HTTP/1.1\r\n
```

---

The fix is to pass the prologue as the argument instead of the format.

Before:

```java
ctx.log(LOGGER, System.Logger.Level.WARNING, "Request failed: " + request.prologue()
        + ", cannot send error response, as response already sent", e);
```

After:
```java
ctx.log(LOGGER, System.Logger.Level.WARNING,
        "Request failed: %s, cannot send error response, as response already sent",
        e, request.prologue());
```

--- a/webserver/webserver/src/main/java/io/helidon/webserver/http/ErrorHandlers.java
+++ b/webserver/webserver/src/main/java/io/helidon/webserver/http/ErrorHandlers.java
@@ -144,8 +144,9 @@ public final class ErrorHandlers {
                                         RoutingResponse response,
                                         RequestException e) {
         if (!response.reset()) {
-            ctx.log(LOGGER, System.Logger.Level.WARNING, "Request failed: " + request.prologue()
-                    + ", cannot send error response, as response already sent", e);
+            ctx.log(LOGGER, System.Logger.Level.WARNING,
+                    "Request failed: %s, cannot send error response, as response already sent",
+                    e, request.prologue());
             throw new CloseConnectionException(
                     "Cannot send response of an error handler, status and headers already written");
         }

